deployer/src/deployer/analyze_pr.py

```diff
@@ -222,20 +233,14 @@ def post_about_flaws(build_directory: Path, **config):
         comments.append((doc, "\n".join(flaws_list)))
 
     def count_flaws(flaws):
+        nonlocal total_flaws
         count = 0
         for flaw in flaws.values():
             count += len(flaw)
+        total_flaws += count
```

Comment:
Instead of adding a side-effect to `count_flaws`, can we just add `flaw_count` to `total_flaws` below?

Reply:
If someone decides to use the function at multiple places then the addition will have to be done after every call. :thinking:

Let's not worry about the future case. I've committed your suggestion.